src/components/notifications/Discord.vue

```diff
@@ -53,13 +53,24 @@
             </div>
         </div>
     </div>
+
+    <div class="mb-3">
+        <label for="discord-disable-url" class="form-label">{{ $t("Disable URL in Notification") }}</label>
+        <div class="form-check form-switch">
+            <input id="discord-disable-url" v-model="$parent.notification.disableUrl" class="form-check-input" type="checkbox">
```

Comment:
`role="switch"` can be quite nice for a11y.

```suggestion
            <input id="discord-disable-url" v-model="$parent.notification.disableUrl" class="form-check-input" type="checkbox" role="switch">
```